OWNERS.md

```diff
@@ -112,6 +112,9 @@ contributors to envoy-setec and relevant Slack channels from:
 * Jose Nino ([junr03](https://github.com/junr03)) (recruiting@junr03.com)
 * Dhi Aurrahman ([dio](https://github.com/dio)) (dio@rockybars.com)
 * Dmitry Rozhkov ([rojkov](https://github.com/rojkov)) (dmitry.rozhkov@intel.com)
+* Michael Rebello ([rebello95](https://github.com/rebello95)) (mrebello@lyft.com)
+* Alan Chiu ([buildbreaker](https://github.com/buildbreaker)) (achiu@lyft.com)
```

Comment:
if nothing else these emails are invalid now

Reply:
Yeah, not sure if they want their personal emails published here. Will leave as-is.